app/controllers/concerns/billable_event_trackable.rb

```diff
@@ -22,6 +22,9 @@ def create_sp_return_log(billable:)
       billable: billable,
       ial: user_ial_context.bill_for_ial_1_or_2,
       issuer: current_sp.issuer,
+      profile_id: current_user&.active_profile&.id,
+      profile_verified_at: current_user&.active_profile&.verified_at,
+      profile_requested_issuer: current_user&.active_profile&.initiating_service_provider,
```

Comment:
First, `current_user` should always be present, so there should be no need for the `&` directly after it.

Second, we are not always returning profile attributes to partners, maybe we could only check for `active_profile` if `ial > 1` ?

```suggestion
      profile_id: current_user.active_profile&.id,
      profile_verified_at: current_user.active_profile&.verified_at,
      profile_requested_issuer: current_user.active_profile&.initiating_service_provider,
```